documentation/getting-started.adoc

```diff
@@ -0,0 +1,245 @@
+== Getting started
+
+=== Prerequisites
+
+Kubernetes or OpenShift cluster are required to deploy Strimzi. Strimzi supports all kind of clusters - from public and
```

Comment:
"...is required..." . Do you have the intention to refer one possible cluster Kubernetes or OpenShift ?